docker-compose.yml

```diff
@@ -114,6 +114,7 @@ services:
       RUST_LOG: firezone_linux_client=trace,wire=trace,connlib_client_shared=trace,firezone_tunnel=trace,connlib_shared=trace,warn
       FIREZONE_API_URL: ws://api:8081
       FIREZONE_ID: D0455FDE-8F65-4960-A778-B934E4E85A5F
+    init: true
```

Comment:
Normally stopping the client takes 10 seconds because Docker times out and SIGKILLs it. Passing `init: true` forwards the equivalent of SIGTERM or SIGINT or whatever and the container exits gracefully in 0.5 seconds.

Reply:
For some reason it didn't work on relay or gateway. Maybe they don't catch Ctrl+C?

Reply:
I think both of those have a shellscript that runs the actual binary.

Reply:
It's because the Linux client explicitly uses [`block_on_ctrl_c()`](https://github.com/firezone/firezone/blob/main/rust/linux-client/src/main.rs#L36).